addons/sourcemod/scripting/l4d_tank_control_eq.sp

```diff
@@ -302,6 +306,8 @@ public chooseTank()
             queuedTankSteamId = "";
         }
         
+        CloseHandle(infectedTeam);
+        CloseHandle(infectedPool);
```

Comment:
Looks like there's another leak in this file on line 301:

```pawn
h_whosHadTank = removeTanksFromPool(h_whosHadTank, teamSteamIds(L4D2Team_Infected));
```

That's another array being created in `teamSteamIds()` that never gets closed.

Might be a good to rename `teamSteamIds` to `createTeamSteamIdsArray`  so it looks more like something that creates a handle?